packages/admin/cms-admin/src/contentScope/Controls.tsx

```diff
@@ -18,34 +19,69 @@ interface ContentScopeControlsProps<S extends ContentScopeInterface = ContentSco
     config?: ContentScopeControlsConfig<S>; // form configuration
 }
 
+export type ContentScopeCombinations = {
+    mapping: string[];
+    grouping?: ContentScopeInterface;
+    values: ContentScopeInterface[];
```

Comment:
What is the idea behind each of this field? Could you please provide an example of scope combinations?

Reply:
mapping is used to get the information about the order of the scope Dimensions.
grouping is the first dimension that is usually used to display the group inside the select
values is an array containing all possible combinations of the remaining dimensions under the grouping.

<details><summary>Example:</summary>

```
[
    {
        "mapping": [
            "domain",
            "language"
        ],
        "grouping": {
            "value": "main"
        },
        "values": [
            {
                "value": "en",
                "label": "English"
            },
            {
                "value": "de",
                "label": "German"
            }
        ]
    },
    {
        "mapping": [
            "domain",
            "language"
        ],
        "grouping": {
            "value": "secondary"
        },
        "values": [
            {
                "value": "en",
                "label": "English"
            },
            {
                "value": "de",
                "label": "German"
            }
        ]
    }
]
```

</details>